src/TLuaInterpreter.cpp

```diff
@@ -5358,7 +5353,7 @@ int TLuaInterpreter::setPopup(lua_State* L)
 
     Host& host = getHostFromLua(L);
     if (_commandList.size() != _hintList.size()) {
-        lua_pushstring(L, "Error: command list size and hint list size do not match cannot create popup");
+        lua_pushstring(L, "setPopup: commands and hints list aren't the same size");
```

Comment:
Technically I think this is a run-time value error (to be handled by a `nil` + error message) rather than a type-eror (where we do throw a `lua_error` AND **abort the calling function/script**!).

Any thoughts from @Mudlet/lua-interface ?

Reply:
Probably a discussion for a new issue? This would then also need to change for the other popup functions (insertPopup, echoPopup)